File: data/data.code.lua.server/AckSceneExit.lua
module("AckSceneExit", package.seeall)


function new()
	local tab_obj = {}
	setmetatable(tab_obj, {__index = AckSceneExit})
	tab_obj:init()
	return tab_obj
end

function init(self)
	self.uid = 0
end


function encode(self)
	local pack = packet.new()
	pack:writeU32(self.uid)
	return pack:encode(Msg.P_ACK_SCENE_EXIT)
end

File: data/data.code.golang.server/ack_scene_exit.go
package proto

import (
	"packet"
)

type AckSceneExit struct {
	Uid                      uint32
}

func (this *AckSceneExit) Encode() []byte {
	pack := packet.NewWriteBuff(64)

	pack.WriteUint32(this.Uid)

	return pack.Encode(P_ACK_SCENE_EXIT)
}
